### PR TITLE
Open ai media streaming sample

### PR DESCRIPTION
**Pull request by v-durgeshs**
Added sample code.
updated rt-client

### CODE REVIEW

**Review by fhaghbin-msft — APPROVED** [review]
Make sure it runs without any errors before merge

**Review by anujissarMS — APPROVED** [review]
LGTM

**Comment by fdalorzo-msft** [comment]
**NIT:** I did a quick `eslint`, I found a few issues, I don't think they are critical, but I thought it was worth sharing it
```
\src\app.ts
   35:43  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   35:53  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   80:51  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
   80:56  error  'res' is defined but never used           @typescript-eslint/no-unused-vars
   80:61  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  107:70  error  Empty block statement                     no-empty

\src\azureOpenAiService.ts
  58:9  error  'configMessage' is never reassigned. Use 'const' instead  prefer-const
```

File: callautomation-az-openai-voice/src/azureOpenAiService.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+const answerPromptSystemTemplate = `You are an AI assistant that helps people find information.`
+
+let realtimeStreaming: LowLevelRTClient;
+
+export async function sendAudioToExternalAi(data: string) {
+    try {
+        const audio = data
+        if (audio) {
+            realtimeStreaming.send({
```
Comment (fdalorzo-msft) [inline callautomation-az-openai-voice/src/azureOpenAiService.ts:21]: I think we need to add an `await` here:
```typescript
await realtimeStreaming.send({
                type: "input_audio_buffer.append",
                audio: audio,
            });
```

File: callautomation-az-openai-voice/src/azureOpenAiService.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            await handleRealtimeMessages();
+        } catch (error) {
+            console.error('Error handling real-time messages:', error);
+        }
+    });
+}
+
+function createConfigMessage(): SessionUpdateMessage {
+
+    let configMessage: SessionUpdateMessage = {
```
Comment (fdalorzo-msft) [inline callautomation-az-openai-voice/src/azureOpenAiService.ts:58]: **NIT**:  maybe we could get rid of  `configMessage`  and return directly

```typescript
return {
        type: "session.update",
        session: {
            instructions: answerPromptSystemTemplate,
            voice: "shimmer",
            input_audio_format: "pcm16",
            output_audio_format: "pcm16",
            turn_detection: {
                type: "server_vad",
            },
            input_audio_transcription: {
                model: "whisper-1"
            }
        }
    };
```

**Review by fdalorzo-msft — APPROVED** [review]

**Pull request merged**